src/workloads/execution/ColumnstoreIndexes.yml

```diff
@@ -0,0 +1,463 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-execution"
+Description: |
+  This workload compares the query performance with columnar indexes to performances without them.
+
+  The workload consists of the following phases and actors:
+    0. Warm up cache.
+    1. Run queries with columnstore indexes.
+    2. Drop columnstore indexes.
+    3. Warm up cache again for regular scans.
+    4. Run queries without columnstore indexes.
+
+Keywords:
+- columnstore
+- analytics
+
+# Defines the database name.
+Database: &db charts-metrics
+
+# Defines the collection name.
+Collection: &Collection events
+
+CountPerActionQuery: &CountPerActionQuery
+  aggregate: *Collection
+  pipeline: [{"$group": {"_id": "$action", o: {$sum: 1}}}, {"$limit": 5000}]
+  allowDiskUse: true
+  cursor: {batchSize: 5000}
+
+TotalTenantCreatedQuery: &TotalTenantCreatedQuery
+  aggregate: *Collection
+  pipeline:
+    [
+      {"$match": {"metadata.target": "cloud"}},
+      {"$group": {"_id": {}, "__alias_0": {"$addToSet": "$metadata.tenant_id"}}},
+      {"$project": {"_id": 0, "__alias_0": {"$size": "$__alias_0"}}},
+      {"$project": {"value": "$__alias_0", "_id": 0}},
+      {"$limit": 5000}
+    ]
+  allowDiskUse: true
```

Comment:
This is the default now, right? Consider just removing this throughout to reduce the noise for the reader.

Reply:
sure, done. Some people wanted to see it explicitly in other PRs. Everyone has different preference. So...

Reply:
Ah! Sorry to flip flop on it! Yes I had the same thought about multi version and that column store wouldn't make sense in that scenario. But I am happy with either - consistency is also valuable.